src/MapfileStyleParser.ts

```diff
@@ -382,33 +387,48 @@ export class MapfileStyleParser implements StyleParser {
     }
 
     const symbolType = mapfileStyle.symbol.type.toLowerCase();
-    const points = mapfileStyle.symbol.points.split(' ').map((item) => parseFloat(item));
-    if (symbolType === 'ellipse') {
-      if (!(points[0] === points[1] && points.length === 2)) {
-        console.warn('Custom ellipse not supported by MarkerSymbolyzer, fallback to "Circle"!');
-      }
-      markSymbolizer.wellKnownName = 'circle' as WellKnownName;
-    } else {
-      if (isSquare(points)) {
-        markSymbolizer.wellKnownName = 'square' as WellKnownName;
+    if (mapfileStyle.symbol.points) {
+      const points = mapfileStyle.symbol.points.split(' ').map((item) => parseFloat(item));
+      if (symbolType === 'ellipse') {
+        if (!(points[0] === points[1] && points.length === 2)) {
+          console.warn('Custom ellipse not supported by MarkerSymbolyzer, fallback to "Circle"!');
```

Comment:
I'm not sure, if the parser should provide such fallbacks, or if this is rather a project specific post processing step

Reply:
It depends on how much gets preserved of the style without fallback. Probably it is not necessary, though nice to have a visible indication and a dummy style to start off post processing.

Reply:
1. I think it depends on the users preferences what should happen if some style-features are not supported. IMHO we should not transform here.
2. This case is kind of special though. As it effectively is a circle. We're back at the discussion [file-similarity vs visual-similarity](https://github.com/geostyler/geostyler/discussions/1451). Cause we basically want to be able to parse forth and back and still receive the same result as the original input.
3. We could introduce `ellipse` as WellKnownName to the geostyler-style. Which would resolve both issues here, right? If there are even more "basic" styles we dont support yet, we can add them, too. The geostyler-style is not bound to any official standard here as it is just an exchange format.

Reply:
@KaiVolland this sounds reasonable, i don't really mind any direction here. There is just that an ellipse in geostyler style is not sufficient to handle all custom vector symbols form mapserver.

Reply:
Im not into mapserver styling at all. But the vector symbols seem to be kind of a special thing of mapserver. They wouldn't be to hard to parse and i can imagine that you could create similar symbols for other styles too (canvas, base64svg, …), but im not sure if this is worth the effort at this point.

Reply:
Yes, we were thinking about handling them as svg. Never got the time though. This pr is basically my hacks i did shortly before the presentation to the customer.

Reply:
I think introducing such a fallback or dummy style to start off post processing would be quite helpful, for instance as `WellKnownName`. This would certainly simplify it for all parsers to deal with such cases. Project specific adjustments can then still be done in post processing. In my opinion, the prerequisite would be that it can then be implemented in all other parsers as well.

Reply:
Has there been a consent on this? I would like to finish this one way or the other if possible. Thanks!

Reply:
I would suggest to not provide a fallback here. So you could follow @KaiVolland's recommendation of either adding the `ellipse` type to our list of WellKnownNames and then create the proper style object, or handle such cases in a post-processing step.